### PR TITLE
Feature/285 generate report of all combinations of the census variables in which each combination gets a table and chart

### PR DESCRIPTION
**Pull request by joebrew**
https://trello.com/c/IDcZewt8/285-generate-report-of-all-combinations-of-the-census-variables-in-which-each-combination-gets-a-table-and-chart

### CODE REVIEW

File: reports/get_stats.rmd [hunk 1]
```diff
@@ -0,0 +1,310 @@
+---
+title: "Youthrex data table script"
```
Comment (benmbrew) [inline reports/get_stats.rmd]: Do you use anything from this script? I know i wrote some of this code, but I can't remember which one is mine.

File: reports/get_stats.rmd [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+library(knitr)
+
+
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, comment=NA, error=FALSE}
+##########
+# read in and finish cleaning census data 
+##########
+dat <- read_feather('data/census_all.feather')
```
Comment (benmbrew) [inline reports/get_stats.rmd]: I don't have a census_all.feather on this branch, only a census.feather. 

When I replaced it with census.feather, it ran, but the markdown just had a few lines and no tables.

But maybe this isnt the script I should be looking at?

File: reports/tables.Rmd [hunk 3]
```diff
@@ -0,0 +1,250 @@
+---
+title: YouthREX tables
```
Comment (benmbrew) [inline reports/tables.Rmd:2]: This is the script I should be looking at right? Why did you make changes to the get_stats script? 
Looked over the code and it looks great. Those are some pretty awesome for loops. 
It's still running, but I'm pretty sure its finishing up. All looks good to me, if you haven't already merged!

**Review by benmbrew — COMMENTED** [review]
Looked over it and left one comment, but it looks awesome!

**Comment by benmbrew** [comment]
Back in your court!

**Comment by benmbrew** [comment]
Annndddd it broke my computer...

**Comment by benmbrew** [comment]
Ok it finished. awesome - thanks for doing this

**Comment by joebrew** [comment]
@benmbrew Indeed, the only "new" code is the tables script. I moved your other exploratory stuff just to have all Rmds in the same place. Merging.

**Pull request merged**